crates/oxc_linter/src/rules/jest/prefer_todo.rs

```diff
@@ -0,0 +1,249 @@
+        // Todo: this punctuation should read from the config
+        formatter.print(b'\'');
+        formatter.print_str(ident.value.as_bytes());
+        formatter.print(b'\'');
+        formatter.print(b')');
+    } else if let Argument::Expression(Expression::TemplateLiteral(temp)) = &expr.arguments[0] {
+        formatter.print(b'`');
+        for q in &temp.quasis {
+            formatter.print_str(q.value.raw.as_bytes());
+        }
+        formatter.print(b'`');
+        formatter.print(b')');
+    }
+
+    (formatter.into_code(), expr.span)
+}
+
+#[test]
+fn tests() {
+    use crate::tester::Tester;
+
+    let pass = vec![
+        ("test()", None),
+        ("test.concurrent()", None),
+        ("test.todo('i need to write this test correct');", None),
+        ("test(obj)", None),
+        ("test.concurrent(obj)", None),
+        ("fit('foo')", None),
+        ("fit.concurrent('foo')", None),
+        ("xit('foo')", None),
+        ("test('foo', 1)", None),
+        ("test('stub', () => expect(1).toBe(1));", None),
+        ("test.concurrent('stub', () => expect(1).toBe(1));", None),
+        (
+            "
+                supportsDone && params.length < test.length
+                    ? done => test(...params, done)
+                    : () => test(...params);
+            ",
+            None,
+        ),
+    ];
+
+    let fail = vec![
+        ("test('i need to write this test');", None),
+        ("test('i need to write this test',);", None),
+        ("test(`i need to write this test`);", None),
+        ("it('foo', function () {})", None),
+        ("it('foo', () => {})", None),
+        ("test.skip('i need to write this test', () => {});", None),
+        ("test.skip('i need to write this test', function() {});", None),
+        ("test[`skip`]('i need to write this test', function() {});", None),
+        ("test[`skip`]('i need to write this test', function() {});", None),
+    ];
+
+    let fix = vec![
+        (
+            "test.skip('i need to write this test');",
+            "test.todo('i need to write this test');",
+            None,
+        ),
+        ("test('i need to write this test',);", "test.todo('i need to write this test',);", None),
+        ("test(`i need to write this test`);", "test.todo(`i need to write this test`);", None),
+        ("it.skip('foo', function () {})", "it.todo('foo')", None),
+        ("it(`i need to write this test`, () => {})", "it.todo(`i need to write this test`)", None),
+        (
+            "test.skip('i need to write this test', () => {});",
+            "test.todo('i need to write this test');",
+            None,
+        ),
+        (
+            "test.skip('i need to write this test', function() {});",
+            "test.todo('i need to write this test');",
+            None,
+        ),
+        // Todo
+        // ("test['skip']('i need to write this test', function() {});", "test['todo']('i need to write this test');", None),
```

Comment:
Is there any suggestions about this case? I didn't find methods how could both replace code and remove code.

Reply:
Not sure if it is helpful, what about continuing match on `MemberExpression::ComputedMemberExpression` or `MemberExpression::StaticMemberExpression` to produce different codes in `build_code `.

Reply:
Thanks too much. It WORKED. I need to spend time to read TC39 Standard.